### DISCUSSION TITLE
What is difference between odds ratio and beta coefficient?

### DISCUSSION THREAD

**Discussion by kaanokay** [MRCIEU/TwoSampleMR#282] [Q&A]
Hi,

First of all, I'd like to thank you for this amazing tool.

I have a question about odds ratio and beta coefficient usage during TwoSampleMR analysis. I have two case-control GWAS data sets. A GWAS data has odds ratio, but other has beta-coefficient. I want to perform bi-directional MR analysis for two datasets. Do I need to convert odds ratio and its standard error to beta coefficient and its standard error for bidirectional analysis? Or, can I use odds ratio for a data and beta coefficient for other data without any conversion? If I make analysis without conversion of odds ratio to beta values, is this a problem? Does it effect my results?

Thank you for your help and interest.
Best regards.

**Comment by mocksu** [comment]
I think you should use beta for both. beta = log(odds ratio)

**Reply by kaanokay** [reply]
@mocksu,

Thank you for your reply. I tried to make analysis using beta coefficient, and obtained better results. Thank you!
